helm/cluster-azure/values.yaml

```diff
@@ -70,9 +70,6 @@ machineDeployments:
 
 sshSSOPublicKey: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIM4cvZ01fLmO9cJbWUj7sfF+NhECgy+Cl0bazSrZX7sU vault-ca@vault.operations.giantswarm.io"
 
-# Used by `cluster-shared` library chart
-includeClusterResourceSet: true
```

Comment:
removing this with the current version of `cluster-shared` dependant chart will break the cluster because the PSP won't be installed, right ?

Reply:
The current implementation

https://github.com/giantswarm/cluster-shared/blob/v0.6.4/helm/cluster-shared/templates/_helpers.tpl#L3

means that the result will always be `true`, even if the value `.Values.includeClusterResourceSet` is false. So the ClusterResourceSet will always be included. There is currently no way to deactivate.

See https://gigantic.slack.com/archives/C01F7T2MNRL/p1676028130499329 for details.

Reply:
Actually the rendering seems ok with your branch, the psp are being rendered 

rendering from main
```
➜ helm template -f ci/ci-values.yaml --kube-version=1.24.0 --api-versions=addons.cluster.x-k8s.io/v1beta1/ClusterResourceSet . | grep psp
  name: release-name-psps
  standard_psps.yml: |
      name: privileged-psp-user
      name: privileged-psp-users
      name: privileged-psp-user
      name: restricted-psp-user
      name: restricted-psp-users
      name: restricted-psp-user
  name: release-name-psps
    name: release-name-psps
```

rendering from this branch
```
➜ helm template -f ci/ci-values.yaml --kube-version=1.24.0 --api-versions=addons.cluster.x-k8s.io/v1beta1/ClusterResourceSet . | grep psp
  name: release-name-psps
  standard_psps.yml: |
      name: privileged-psp-user
      name: privileged-psp-users
      name: privileged-psp-user
      name: restricted-psp-user
      name: restricted-psp-users
      name: restricted-psp-user
  name: release-name-psps
    name: release-name-psps

```

Reply:
yeah, sorry was having lunch :)

Reply:
default value is true BTW

https://github.com/giantswarm/cluster-shared/blob/main/helm/cluster-shared/templates/_helpers.tpl#L3